tests/transforms/test_GaussianBlur.py

```diff
@@ -9,6 +9,6 @@ class TestGaussianBlur(unittest.TestCase):
     def test_on_pil_image(self):
         for w in range(1, 100):
             for h in range(1, 100):
-                gaussian_blur = GaussianBlur(w * 0.1)
+                gaussian_blur = GaussianBlur()
```

Comment:
Could you add an additional test that checks that the old parameters still work and show the deprecation warning? There are some other tests in the codebase that check for deprecation warnings that you can copy.